adagucserverEC/CDBAdapterPostgreSQL.cpp

```diff
@@ -1080,3 +1084,86 @@ int CDBAdapterPostgreSQL::addFilesToDataBase() {
 #endif
   return 0;
 }
+
+int CDBAdapterPostgreSQL::storeLayerMetadata(const char *datasetName, const char *layerName, const char *metadataKey, const char *metadataBlob) {
```

Comment:
Just so I understand correctly:
The table doesn't have seperate columns for the different metadata bits, but instead they are stored in seperate rows, identified by `metadataKey`?

These rows are stored one by one, but always queried together, right?

Reply:
Yes

Reply:
Is there a particular reason for this? Does it make the logic/code easier?

Reply:
Yes, metadata for the layer struct is collected at different places and then immediately stored in the table. 

This gives the opportunity to only update the dimensions and leave the projection, styles and layermetadata alone.